annonatate/flaskserver.py

```diff
@@ -24,6 +23,11 @@
 import os
 #import time
 
+import image as anno
+import search as anno
+
+import pdb
```

Comment:
Might want to comment this out. I am also not that familiar with what this does but it might be equivalent to setting app.run(debug=True) at the end of this script.

Reply:
Removed